.github/workflows/mypy-generic.yml

```diff
@@ -1,22 +0,0 @@
-# This is needed for GitHub Actions for the "Waiting for status to be reported" problem,
```

Comment:
Why is this no longer needed?

Reply:
This is because we are using a wrong required action in our setting. Once we switched to `mypy (3.8)` (we already did), the generic yml is no longer needed.